packages/terra-table/src/TableRows.jsx

```diff
@@ -23,7 +23,7 @@ const defaultProps = {
 };
 
 function cloneChildItems(children, onClick, onKeyDown) {
-  return children.map((child) => {
+  return React.Children.map(children, (child) => {
```

Comment:
An alternative is to mark children as required and or default children to an empty array
```
const defaultProps = {
  children: [],
  onClick: undefined,
  onKeyDown: undefined,
};
```